data/participants.yml

```diff
@@ -1244,3 +1244,36 @@ participants:
 - name: "ZTE"
   link: "http://www.zte.com.cn/global/"
   logo: "commons-logos/zte.png"
+- name: "Momox"
+  link: "https://momox.biz/"
+  logo: "commons-logos/momox.png"
+- name: "Federal Reserve"
+  link: "https://www.federalreserve.gov/"
+  logo: "commons-logos/federalreserve.svg"
+- name: "Fifth Third Bank"
+  link: "https://www.53.com"
+  logo: "commons-logos/fifththirdbank.svg"
+- name: "Intac"
+  link: "http://intaclatam.com/"
+  logo: "commons-logos/intac.png"
+- name: "TYG Peru"
+  link: "www.tygperu.com"
```

Comment:
The protocol-less URL is not in line with the rest of them in the yaml and creates an invalid relative link. This is something that I used to manually fix now and then before. Link validation can be done by middleman when generating the site, but I'd prefer adding a simple check like [this](https://github.com/jiri-fiala/commons.openshift.org/commit/752ce91644922ab4e5b07aceff942487a59f1753) to keep the yaml file clean. What do you think, @wgordon17?